### DISCUSSION TITLE
Upload media that is not bound to a model

### DISCUSSION THREAD

**Discussion by robbens** [spatie/laravel-medialibrary#2034] [Q&A]
Is it possible to upload media without having it attached to a model? 
I have a pivot table, `mediables`, which uses a polymorphic relationship so that I can use the same media for multiple models.

Maybe a fork is the only way?

Thanks!

**Comment by johnchque** [comment]
I also would like to know. AFAIK, the table that comes with the package does not allow the Model fields to be empty. Any workaround for this?
Thanks! :)
